content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -0,0 +1,202 @@
+
+DVC can use multiple threads to upload files (4 per CPU core by default). You
+can set the number with `--jobs` (`-j`). Please note that increases in
+performance also depend on the connection bandwidth and remote configurations.
+
+DVC has a mechanism called the [run-cache] that stores the artifacts from
+intermediate stages. For example, if there is an intermediate step that applies
+data-augmentation on your dataset and you would like to push these artifacts as
+well as the end products of the experiments, you can use `--run-cache` flag to
+push all of these to the DVC remote. `--run-cache` flag pushes all artifacts
+referenced in `dvc.lock` file.
+
+[run-cache]: /doc/user-guide/project-structure/internal-files#run-cache
+
+## Listing experiments remotely
+
+In order to list experiments in a DVC project, you can use the `dvc exp list`
+command. With no command line options, it lists the experiments in the current
+project.
+
+You can supply a Git remote name to list the experiments:
+
+```dvc
+$ dvc exp list origin
+main:
+	cnn-128
+	cnn-32
+	cnn-64
+	cnn-96
+```
+
+Note that by default this only lists experiments derived from the current commit
+(local `HEAD` or default remote branch). You can list all the experiments
+(derived from from every branch and commit) with the `--all` option:
+
+```dvc
+$ dvc exp list origin --all
+0b5bedd:
+	exp-9edbe
+0f73830:
+	exp-280e9
+	exp-4cd96
+	...
+main:
+	cnn-128
+	...
+```
+
+When you don't need to see the parent commits, you can list experiment names
+only, with `--names-only`:
+
+```dvc
+$ dvc exp list origin --names-only
+cnn-128
+cnn-32
+cnn-64
+cnn-96
+```
+
+## Downloading experiments from remotes
+
+When you clone a DVC repository, it doesn't fetch any experiments by default. In
+order to get them, use `dvc exp pull` (with the Git remote and the experiment
+name), for example:
+
+```dvc
+$ dvc exp pull origin cnn-64
+```
+
+This pulls all the necessary files from both remotes. Again, you need to have
+both of these configured (see this
+[earlier section](#prepare-remotes-to-share-experiments)).
+
+You can specify a remote to pull from with `--remote` (`-r`).
+
+DVC can use multiple threads to download files (4 per CPU core typically). You
+can set the number with `--jobs` (`-j`).
+
+If an experiment being pulled already exists in the local project, DVC won't
+overwrite it unless you supply `--force`.
+
+### Pulling all experiments
+
+You can create a loop to pull all experiments from `origin` (Git remote) like
+this:
+
+```dvc
+$ dvc exp list --all --names-only | while read -r expname ; do \
+    dvc exp pull origin ${expname} \
+done
+```
+
+## Example: Creating a directory for an experiment
+
+A good way to isolate experiments is to create a separate home directory for
+each one.
+
+> Another alternative is to use `dvc exp apply` and `dvc exp branch`, but here
+> we'll see how to use `dvc exp pull` to copy an experiment.
```

Comment:
Interesting idea!

I made that last sentence into a note but it's still not clear what that comment is about.

Reply:
Also not 100% sure how this example is about sharing experiments.

Reply:
It's possible to use `dvc exp branch` to create a branch and share it via Git but here we use only DVC commands to share a particular experiment in a separate directory. 

It's a use case to share an experiment by creating a separate directory containing all relevant artifacts. The user can extrapolate the idea to use SSH/rsync or some other means to share an experiment quickly, without setting up a DVC remote.

Reply:
> It's possible to use dvc exp branch to create a branch and share it via Git

I see. That may be a more clear example here actually? Since it's directly about sharing. In fact could be an H2 section since `exp branch` isn't covered in the guide yet.

> share an experiment by creating a separate directory containing all relevant artifacts

Again, I like the idea a lot actually, very creative. But I'm not seeing that as a way of sharing (the extrapolation part would need to be described explicitly). Seems to me like it's more of a way to isolate an experiment in a separate environment (folder), rather. Maybe we can put it in another Exps guide (I'm still working on exploding the long index ⏳).

Reply:
> still working on exploding the long index

See #2654.

Reply:
Thank you :) I assumed the _isolation_ part is the first step to share an experiment. This may be a use case maybe. I'm just trying to bring together all the pieces we've told above for another example.